app/Http/Controllers/Campaign/MemberController.php

```diff
@@ -29,11 +30,15 @@ public function __construct(MemberService $memberService)
      * Switch to a member of the campaign
      * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
      */
-    public function switch(CampaignUser $campaignUser)
+    public function switch(CampaignUser $campaignUser, Entity $entity)
```

Comment:
Please set the entity's default to null, so it's easier to read (it's an optional parameter) :)
`Entity $entity = null`